### ISSUE TITLE
Mail App Not Available

### ISSUE THREAD

**Issue by kreativmind** [owncloud/mail#1642]
How do I add the mail app to my self-hosted version of ownCloud? I have tried to click the plus but nothing shows up except a list of enabled and disabled apps. I have uploaded the folders for the mail app and tried to check the list again but it doesn't ever show up. How do I add the mail app? Thanks

**Comment by kreativmind** [comment]
Anyone...?

**Comment by RealRancor** [comment]
See: https://central.owncloud.org/t/no-apps-listed-no-internet-connection-curl-error-60-message/813

Please note its higly discouraged to use an issue tracker of an app as a support forum

**Comment by kreativmind** [comment]
Will check it out...in regards to support forum....I was told to use this by Frank K. Thanks.

**Comment by RealRancor** [comment]
Hi,

that pointer was mostly wrong :-) If only "Enabled" and "Not Enabled" is shown then its mostly a configuration / environmental issue rather than an app issue.

**Comment by kreativmind** [comment]
Yes only Enabled and Not Enabled shows up. I have installed Nextcloud on the same server as well and it's working flawlessly. Definitely liking NextCloud much better.

**Comment by RealRancor** [comment]
That shouldn't be the topic here ;-)

**Comment by kreativmind** [comment]
Fixed the reply.
